### ISSUE TITLE
Swing Interface

### ISSUE THREAD

**Issue by posenato** [jrtom/jung#81]
Hi,

some years ago I developed a simple graph program using Jung 2.0.1 for showing some new graphs algorithms. The program allows a user to draw a graph, edit node/edge properties and run some checking algorithms. 

I would like to update the program to Jung 2.1.1. 
I checked the notes of 2.1.0 version and changed interfaces/library accordingly, but this is not sufficient to have the update. 
It seems that the Swing interface has been changed in other aspects. 
Is there a guide/tutorial/howto about the new visualization module architecture of the library in order to make easier the update of programs based on the 2.0.1 visualization version?

Thanks,
Roberto

**Comment by jrtom** [comment]
@posenato can you be more specific about what problems you're encountering?

The release notes (https://github.com/jrtom/jung/releases) list the changes made in some detail.

Please note that a more comprehensive overhaul (for JUNG 3.0) is underway, so documentation (and other) efforts are likely to be focused on that work.

**Comment by posenato** [comment]
Hi Joshua,

I made the update of my software considering all your release notes about 2.1.* version.
At run time, the application presented some exceptions about some not defined objects (relative to Swing library).
I am sorry, I don't remember the kind of exception and I have already restored the working version of my software. I cannot give more details. 

Considering your news about JUNG 3.0, I wait for JUNG 3.0 release before trying to update again.
Thanks for your work. 

Roberto
